app/src/main/java/com/alexbar/layoutbasic/movies_app/nav/BottomNavigationGraph.kt

```diff
@@ -4,20 +4,38 @@ import androidx.compose.foundation.background
 import androidx.compose.foundation.layout.Box
 import androidx.compose.foundation.layout.fillMaxSize
 import androidx.compose.runtime.Composable
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
 import androidx.compose.ui.Modifier
 import androidx.navigation.NavHostController
 import androidx.navigation.compose.NavHost
 import androidx.navigation.compose.composable
+import androidx.navigation.navArgument
+import androidx.navigation.navDeepLink
 import com.alexbar.layoutbasic.movies_app.model.Media
 import com.alexbar.layoutbasic.movies_app.screen.detail.MediaDetailScreen
 import com.alexbar.layoutbasic.movies_app.screen.favorites.FavoritesMediaScreen
+import com.alexbar.layoutbasic.movies_app.screen.login.LoginScreen
 import com.alexbar.layoutbasic.movies_app.screen.mix.MixMediaScreen
 import com.alexbar.layoutbasic.movies_app.screen.trending.TrendingMediaScreen
+import com.alexbar.layoutbasic.movies_app.utils.getImageBackdropCompleteUrl
+import com.alexbar.layoutbasic.movies_app.utils.getImagePosterCompleteUrl
 import com.alexbar.layoutbasic.movies_app.utils.getVoteAverageFormatted
 import com.alexbar.layoutbasic.ui.theme.AppBackground
+import java.net.URLEncoder
+import java.nio.charset.StandardCharsets
+
+private fun navigateToMediaDetail(navController: NavHostController, media: Media) {
```

Comment:
this is just a suggestion just for having an organized code, your code is good as well :
```
private fun navigateToMediaDetail(navController: NavHostController, media: Media) {
    val action = BottomNavigationGraphDirections.actionDetailMediaScreen(
        title = media.title ?: media.name ?: "",
        description = media.description ?: "",
        voteAverage = media.getVoteAverageFormatted(),
        imageBackdropUrl = media.getImageBackdropCompleteUrl(),
        imagePosterUrl = media.getImagePosterCompleteUrl()
    )
    navController.navigate(action)
}
```